atomisticparsers/gromacs/parser.py

```diff
@@ -115,11 +134,11 @@ def str_to_step_info(val_in):
         thermo_quantities = [
             Quantity(
                 'energies',
-                r'Energies \(kJ/mol\)\s*([\s\S]+?)\n\n',
+                r'Energies \(kJ/mol\)\s*([\s\S]+?)\nD.* step.* load imb.*|\n\n',
                 str_operation=str_to_energies, convert=False),
             Quantity(
                 'step_info',
-                r'(Step.+\n[\d\.\- ]+)',
+                r'[\r\n]\s*(Step.+\n[\d\.\- ]+)',
```

Comment:
can we perhaps make [\r\n] as universal regex for line breaks

Reply:
done.